### ISSUE TITLE
Integration Tests Not Working

### ISSUE THREAD

**Issue by blgrossMS** [KhronosGroup/UnityGLTF#38]
The integration tests are not running properly. I am on version 2017.1.0f3 of Unity. The error I get is:

```
NullReferenceException: Object reference not set to an instance of an object
UnityTest.GuiHelper.GetConsoleErrorPause () (at Assets/UnityTestTools/IntegrationTestsFramework/TestRunner/Editor/GuiHelper.cs:13)
UnityTest.IntegrationTestsRunnerWindow+RunnerCallback..ctor (UnityTest.IntegrationTestsRunnerWindow window) (at Assets/UnityTestTools/IntegrationTestsFramework/TestRunner/Editor/IntegrationTestsRunnerWindow.cs:499)
UnityTest.IntegrationTestsRunnerWindow.Update () (at Assets/UnityTestTools/IntegrationTestsFramework/TestRunner/Editor/IntegrationTestsRunnerWindow.cs:229)
System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) (at /Users/builduser/buildslave/mono/build/mcs/class/corlib/System.Reflection/MonoMethod.cs:305)
Rethrow as TargetInvocationException: Exception has been thrown by the target of an invocation.
System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) (at /Users/builduser/buildslave/mono/build/mcs/class/corlib/System.Reflection/MonoMethod.cs:313)
System.Reflection.MethodBase.Invoke (System.Object obj, System.Object[] parameters) (at /Users/builduser/buildslave/mono/build/mcs/class/referencesource/mscorlib/system/reflection/methodbase.cs:229)
UnityEditor.HostView.Invoke (System.String methodName, System.Object obj) (at C:/buildslave/unity/build/Editor/Mono/HostView.cs:272)
UnityEditor.HostView.Invoke (System.String methodName) (at C:/buildslave/unity/build/Editor/Mono/HostView.cs:265)
UnityEditor.HostView.SendUpdate () (at C:/buildslave/unity/build/Editor/Mono/HostView.cs:341)
UnityEditor.EditorApplication.Internal_CallUpdateFunctions () (at C:/buildslave/unity/build/Editor/Mono/EditorApplication.cs:109)

```

**Comment by robertlong** [comment]
We shouldn't be using UnityTestTools anymore. They were deprecated in 5.6 and I believe they are broken in 2017.1

https://bitbucket.org/Unity-Technologies/unitytesttools/

I'll migrate everything over to the new built in test runner.

**Comment by robertlong** [comment]
When we do this we should make sure that the integration tests run through all the latest glTF sample models. The models we are testing currently are not comprehensive.

**Comment by ericob** [comment]
This should be fixed in #196 @blgrossMS
